lib/srv/regular/sshserver_test.go

```diff
@@ -1542,6 +1543,61 @@ func TestGlobalRequestRecordingProxy(t *testing.T) {
 	require.True(t, response)
 }
 
+// TestSessionTracker tests session tracker lifecycle
+func TestSessionTracker(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	f := newFixture(t)
+
+	se, err := f.ssh.clt.NewSession()
+	require.NoError(t, err)
+	t.Cleanup(func() { se.Close() })
+
+	// start interactive SSH session (new shell):
+	err = se.Shell()
+	require.NoError(t, err)
+
+	// session tracker should be created
+	var tracker types.SessionTracker
+	condition := func() bool {
+		trackers, err := f.testSrv.Auth().GetActiveSessionTrackers(ctx)
+		require.NoError(t, err)
+
+		if len(trackers) == 1 {
+			tracker = trackers[0]
+			return true
+		}
+		return false
+	}
+	require.Eventually(t, condition, time.Second*5, time.Second)
+
+	// Advance the clock to trigger the session tracker expiration to be extended
+	f.clock.Advance(defaults.SessionTrackerExpirationUpdateInterval)
+
+	// The session's expiration should be udpated
+	condition = func() bool {
+		updatedTracker, err := f.testSrv.Auth().GetSessionTracker(ctx, tracker.GetSessionID())
+		require.NoError(t, err)
+		return updatedTracker.Expiry().After(tracker.Expiry())
+	}
+	require.Eventually(t, condition, time.Second*5, time.Millisecond*1000)
+
+	// Close the session from the client side
+	err = se.Close()
+	require.NoError(t, err)
+
+	// Wait for session to close in the background
+	time.Sleep(defaults.SessionIdlePeriod)
+
+	// once the session is closed, the tracker should expire.
+	condition = func() bool {
+		expiredTracker, err := f.testSrv.Auth().GetSessionTracker(ctx, tracker.GetSessionID())
+		require.NoError(t, err)
+		return time.Now().After(expiredTracker.Expiry())
+	}
+	require.Eventually(t, condition, time.Second*5, time.Millisecond*100)
```

Comment:
```suggestion
	require.Eventually(t, trackerExpired, time.Second*5, time.Millisecond*100)
```

Use a more descriptive name than condition so readers don't have to read the logic in condition to figure out what it's checking for.